cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
I ran the test you posted [here](https://github.com/cydhaselton/coreclr/issues/2#issuecomment-273887631) for `HAVE_COMPATIBLE_EXP_EXITCODE` on-device; it's correct for Android API level 22 but could need testing for other levels, if possible